### PR TITLE
doc(CHANGELOG.md): add changelog entry for beta2

### PR DESCRIPTION
**Pull request by arschles**
This patch also adds the changelog generation script that was used to generate the document herein.

### CODE REVIEW

**Comment by bacongobbler** [comment]
should we merge #229 and #230 into one PR? :)

File: _scripts/generate-changelog.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# vim: et tw=0 sw=4 ts=4
+
+REPOROOT=${REPOROOT:=https://github.com/deis/controller}
```
Comment (mboersma) [inline _scripts/generate-changelog.sh:4]: This should be "charts" or the Markdown links will be wrong.

**Comment by mboersma** [comment]
> should we merge #229 and #230 into one PR? :)

Between the two CHANGELOG.md files we have a complete history of deis/charts now, so yes. I'll cherry-pick from this one into #229.

**Comment by arschles** [comment]
@mboersma thanks! I didn't look for any other PRs first. Closing this in favor of #229 (but leaving the branch alive)

**Pull request closed without merging**